src/projects/project-pull-requests.html

```diff
@@ -125,16 +125,30 @@
         hideProjectButton: {
           type: String,
           value: ''
+        },
+        getFields: {
+          type: Function,
+          value: function() {
+            return function(e) {
```

Comment:
Would getSearchFields be better?
The name should be the same for search-behaviour projects and pr's.